core/src/main/java/io/aiven/klaw/config/ManageDatabase.java

```diff
@@ -181,6 +182,8 @@ private void loadStaticDataToDb() throws KlawException {
           defaultDataService.getDefaultTenant(KwConstants.DEFAULT_TENANT_ID));
     }
 
```

Comment:
can we move the method loadDb() to OnStartup service ?

Reply:
Was referring to removing of InitializingBean interface from ManageDatabase, and invoke the loadDb() method from OnStartup service, but we can move it to a different PR. It is not related here